Libraries/ToTripleSlashPorter.cs

```diff
@@ -186,9 +186,8 @@ private void AddSymbolLocationsToResolvedLocations(ProjectInformation projectInf
                     Log.Info(false, $"Symbol '{symbol.Name}' not found in locations of project '{path}'.");
                     if (n < symbol.Locations.Count())
                     {
-                        Log.Info(false, " Trying the next location...");
+                        Log.Info(true, " Trying the next location...");
                     }
```

Comment:
What's the value in making these each on their own line? As is, it will just keep showing horizontally like a progress indicator, which I'd prefer here. If we do go to newlines, then we can omit the leading space at the beginning of the message.

Reply:
I removed the Console.WriteLine() below (to use the new post/buffer scheme) so I changed this bool to true so we kept the same output as before.

Reply:
Ah; I overlooked the WriteLine below. Got it.